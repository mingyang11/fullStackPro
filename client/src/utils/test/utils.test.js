test('should ', () => {
  expect(2).toBe(2)
})
test('shouldww ', () => {
  expect(3).toMatchSnapshot()
})
