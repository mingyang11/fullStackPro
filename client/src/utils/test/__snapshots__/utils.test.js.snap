// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shouldww  1`] = `3`;
